### PR TITLE
manifest: forgot to add jc

### PR DESCRIPTION
**Pull request by mralusw**
Didn't quite realize manifest has to be updated too...

### CODE REVIEW

**Comment by philclifford** [comment]
No worries. I added it while correcting the docs so this may be superfluous. If I make it home in time I can merge this and revise my PR if needed.

**Review by philclifford — APPROVED** [review]

**Comment by philclifford** [comment]
Actually I can't merge it because a test status is required but the test doesn't run on changes only to manifest.  That needs fixing.  Meantime either @flexiondotorg needs to override the requirement or you might be able to provoke a test run by amending the commit to include a change in `packages/jc` (e.g. adding a comment or two from your excellent explanation of the string variable manipulations to help any future contributors improving their `bash` to read )

**Comment by mralusw** [comment]
I've copied the anchoring to the rightmost dquote from #934 (has that PR been missed, BTW?)

**Pull request merged**
